bitmap-team-TeamName/model/bmp-constructor.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const fileReadHelper = require(`${__dirname}/../lib/bmp-file-helper.js`);
+module.exports = exports = {};
+
+var bmObj = {};
+
+exports.makeBitMap = function(data) {
+  bmObj = new BitMap(data);
+  bmObj.prepareArray();
+  selectTransformation();
+  bmObj.packageArray();
+  fileReadHelper.newBitMap(bmObj.buffer);
+};
+
+function BitMap(data) {
+  this.type = data.toString('utf-8', 0, 2);
+  this.totalFileSize = data.readInt32LE(2);
+  this.arrayLoc = data.readInt32LE(10);
+  this.width = data.readInt32LE(18);
+  this.height = data.readInt32LE(22);
+  this.numColorPanes = data.readInt32LE(26);
+  this.bitsPerPixel = data.readInt32LE(28);
+  this.horizRes = data.readInt32LE(38);
+  this.verticRes = data.readInt32LE(42);
+  this.singles = data.toString('hex', 66, 1090).split('');
+  this.buffer = data;
```

Comment:
@npisciotti1 @radenska @sneakyneeks2 @darms great job including the raw buffer as part of your constructor!!  generally speaking, this constructor looks great - lots of good and usable metadata extracted from the bitmap.  i also really like the `pairs`, `packaged`, and `transformed` arrays that are used to collect data